doc/transform_methods/hansenlaw.rst

```diff
@@ -121,6 +121,19 @@ If you would like to access the Hansen-Law algorithm directly (to transform a
 right-side half-image), you can use :func:`abel.hansenlaw.hansenlaw_transform`.
 
 
+
+Tips
+----
+
+`hansenlaw` tends to perform better with images of large size :math:`n \gt 1001` pixel width. For smaller images the angular_integration (speed) profile may look better if sub-pixel sampling is used via: ::
+
+    transform_options=dict(dr=0.1)
```

Comment:
Ok, we are agreed on this, `dr` is only required for the `angular_integration`. My analysis is unique in that the measured VMI may have any pixel size. I cheat `transform_options` to ensure the intensity scaling is independent of the image size. Agreed, there is little reason for users to employ this.  
(1) Transform comment removed.
(2) `dr=0.5` is fine, `0.2` better, all depends on the image structure, and illusion of improvement due to the interpolation spline.